ansible_collections/arista/avd/roles/eos_validate_state/tasks/main.yml

```diff
@@ -40,34 +40,33 @@
   tags:
     - always
```

Comment:
why set tags when we have `always` everywhere?

Reply:
it was done like this initially - I can remove them probably

Reply:
tags would sstill  be useful for `--skip-tags`
but we have another point to review all the tags strategy in AVD